### PR TITLE
Update potentialVendors query to include organizations created by admins

### PR DESCRIPTION
**Pull request by kewitz**

### CODE REVIEW

File: server/graphql/v2/object/Host.js [hunk 1]
```diff
@@ -1091,23 +1091,36 @@ export const GraphQLHost = new GraphQLObjectType({
 
           const pageQuery = `
                 WITH hostadmins AS (
-                  SELECT "MemberCollectiveId" FROM "Members" WHERE "CollectiveId" = :hostid AND "deletedAt" IS NULL AND role = 'ADMIN'
-                ), orgs AS (
-                  SELECT c.id, c.slug,ARRAY_AGG(DISTINCT m."MemberCollectiveId") as "admins", ARRAY_AGG(DISTINCT t."HostCollectiveId") as hosts
+                  SELECT m."MemberCollectiveId", m."CreatedByUserId" as "UserId"
```
Comment (Betree) [inline server/graphql/v2/object/Host.js:1094]: `m."CreatedByUserId"` cannot safely be interpreted as "User ID of the member"; it contains the ID of the user who created/invited the member, not the User ID itself. 

I suppose your intention was to write:

```suggestion
                  SELECT m."MemberCollectiveId", u."id" as "UserId"
```

File: server/graphql/v2/object/Host.js [hunk 2]
```diff
@@ -1091,23 +1091,36 @@ export const GraphQLHost = new GraphQLObjectType({
...
-                  "admins" <@ ARRAY(SELECT "MemberCollectiveId" FROM hostadmins)
-                  AND "hosts" = ARRAY[:hostid]
+                  (
+                    o."admins" <@ ARRAY(SELECT "MemberCollectiveId" FROM hostadmins)
+                      OR (
+                        o."CreatedByUserId" IN (
+                        SELECT "UserId"
+                        FROM hostadmins
+                        )
+                        AND o."admins" = ARRAY[null]::INTEGER[]
```
Comment (Betree) [inline server/graphql/v2/object/Host.js:1120]: I'm not convinced by this `ARRAY[null]::INTEGER[]`, organizations created via added funds do have an admin, it's just that it's an invited account (that often doesn't get confirmed)

File: server/graphql/v2/object/Host.js (hunk 1 above)
Comment (kewitz) [reply server/graphql/v2/object/Host.js:1094]: Thanks! Nice catch, this was here from the first attempt.

File: server/graphql/v2/object/Host.js (hunk 2 above)
Comment (kewitz) [reply server/graphql/v2/object/Host.js:1120]: Yes. I was working with the idea that I didn't want to block Organizations that were never activated from becoming Vendors. (Assuming the invited admin never had the intention to join).

**Pull request merged**